lib/mpl_toolkits/axisartist/floating_axes.py

```diff
@@ -351,12 +350,8 @@ def adjust_axes_lim(self):
         self.set_ylim(ymin-dy, ymax+dy)
 
 
-@functools.lru_cache(None)
-def floatingaxes_class_factory(axes_class):
-    return type("Floating %s" % axes_class.__name__,
-                (FloatingAxesBase, axes_class), {})
-
-
+floatingaxes_class_factory = cbook._make_class_factory(
+    FloatingAxesBase, "Floating {}")
```

Comment:
I guess this is an old typo, but surely this should not have a space in the name?

Reply:
removed the space.